03_launch_mgmt_cluster.sh

```diff
@@ -139,6 +139,8 @@ function update_images(){
 function launch_ironic() {
   pushd "${BMOPATH}"
 
+  inspector_default=$(grep USE_IRONIC_INSPECTOR "${BMOPATH}/ironic-deployment/default/ironic_bmo_configmap.env" || true)
```

Comment:
It would be great to have it `^IRONIC_USE_INSPECTOR`, in case someone has a comment line with IRONIC_USE_INSPECTOR on it.